applications/tari_base_node/windows/README.md

```diff
@@ -0,0 +1,60 @@
+# Tari Base Node and Wallet Runtime Instructions
+
+## Pre-requisites 
+
+- SQLite:
+  - Download 32bit/64bit Precompiled Binaries for Windows for [SQL Lite](https://www.sqlite.org/index.html). 
+  - Extract to local path, e.g. `%USERPROFILE%\.sqlite`.
```

Comment:
Dot folders are problematic to create in some versions of windows (i.e you have to do it through the mkdir command in the cmd)

Reply:
I think we can leave it like that, it is just an example location.